testing/cachefilesd-inotify/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Jake Buchholz <tomalok@gmail.com>
+# Maintainer: Jake Buchholz <tomalok@gmail.com>
+pkgname=cachefilesd-inotify
+pkgver=0.11.0
+pkgrel=0
+pkgdesc="Userspace daemon acting as a backend for FS-Cache (uses INOTIFY instead of DNOTIFY)"
+url="https://gitlab.com/tomalok/cachefilesd-inotify"
+arch="all"
+license="GPL-2.0-or-later"
+depends="attr"
+makedepends="rpm file"
+provides="cachefilesd"
```

Comment:
I think provides needs to be like `provides="cachefilesd=$pkgver-r$pkgrel`

Reply:
https://wiki.alpinelinux.org/wiki/APKBUILD_Reference#provides doesn't mention `$pkgrel`, I'm not sure behavior beyond `$pkgver` is defined?  (Releases may not necessarily be kept in sync...)

I've updated to add the `$pkgver`.